opentracing-api/src/main/java/io/opentracing/tag/ShortTag.java

```diff
@@ -13,6 +13,7 @@
  */
 package io.opentracing.tag;
 
+@Deprecated
```

Comment:
Your means, we do NOT encourage using `ShortTag` in the future. If so, we should add a comment to explain this.